src/ECharts/Type.purs

```diff
@@ -0,0 +1,23 @@
+module ECharts.Type where
+
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+
+data ChartType = Line | Bar | Scatter | Candlestick | Pie | Radar
```

Comment:
This is not used anywhere, is it, since it's local to the generation of the series???